Index.html

```diff
@@ -18,6 +18,7 @@
 </nav>
   </header>
   <main>
+
```

Comment:
- [Optional] I noticed there is a space to the right of your page, Kindly remove that, please. You can do that by adding `margin: 0;` to the `body`.
![4](https://user-images.githubusercontent.com/94551484/198818431-5b401159-fe0b-434d-9529-53fb05d32173.png)